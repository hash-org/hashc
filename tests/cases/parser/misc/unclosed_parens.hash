// run=fail, stage=parse

t := () => {
    ((a)
};
