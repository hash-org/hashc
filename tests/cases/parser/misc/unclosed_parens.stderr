error: encountered unclosed delimiter `(`, add a `)` after the inner expression
 --> $DIR/unclosed_parens.hash:4:5
3 |   t := () => {
4 |       ((a)
  |       ^ here
5 |   };
